### DISCUSSION TITLE
Use lit-element in MDX

### DISCUSSION THREAD

**Discussion by dmondev** [storybookjs/storybook#12706] [Help] [answered]
Hi, is there an example of creating a story in MDX using lit-element / web component ?

I see that https://github.com/storybookjs/storybook/tree/next/examples/web-components-kitchen-sink/stories is using CSF format.

Thanks !

**Comment by shilman** [comment, marked as answer]
this isn't enough? https://raw.githubusercontent.com/storybookjs/storybook/next/examples/web-components-kitchen-sink/stories/addon-docs.stories.mdx

**Reply by dmondev** [reply]
@shilman, Thanks for the quick response, and sorry for starting the thread. 

Yes, that is more than enough! 

I feel embarrassed to say that I missed that because I opened the file in github and didn't look into the raw source :)

**Reply by DannyClaydenChambers** [reply]
404: Not Found

**Reply by shilman** [reply]
https://raw.githubusercontent.com/storybookjs/storybook/next/examples/web-components-kitchen-sink/src/stories/addons/docs/dynamic-source.stories.mdx

**Reply by andy4577** [reply]
Can anyone update links, please?
